File: 1768-MergeStringsAlternately/1768-MergeStringsAlternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        len1 = len(word1)
        len2 = len(word2)
        
        minLength = min(len1, len2)
        result = ''
        for i in range(minLength):
            result += word1[i] + word2[i]
        
        return result + (word1[i+1:] if len1 > len2 else word2[i+1:])

File: 1768-MergeStringsAlternately/1768-MergeStringsAlternately.php
class Solution {

    /**
     * @param String $word1
     * @param String $word2
     * @return String
     */
    function mergeAlternately($word1, $word2) {
        $res = "";
        for ($i = 0; $i < max(strlen($word1), strlen($word2)); $i++) {
            $res .= ($word1[$i] ?? '') . ($word2[$i] ?? '');
        }
        return $res;
    }
}